src/backend/LamaAPI/Application/Sharing/Commands/SharePhoto/SharePhotoCommandHandler.cs

```diff
@@ -27,18 +28,41 @@ public SharePhotoCommandHandler(IMapper mapper, IApplicationDbContext context)
 		}
 
 		public async Task<SharedEmailsListDTO> Handle(SharePhotoCommand request, CancellationToken cancellationToken)
+		{
+			SharedPhoto sharedPhoto = await SharePhotoAsync(request, cancellationToken);
+
+			await NotifyUserAsync(sharedPhoto.SharedWithUserEmail, cancellationToken);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5221288)